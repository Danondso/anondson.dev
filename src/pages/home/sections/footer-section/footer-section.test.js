import React from 'react';
import renderer from 'react-test-renderer';
import FooterSection from './footer-section';

describe('Footer Section Test Suite', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<FooterSection />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
