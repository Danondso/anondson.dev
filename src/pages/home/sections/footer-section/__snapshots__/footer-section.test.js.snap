// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Section Test Suite renders correctly 1`] = `
<section
  alt="Colored background containing contact and footer info"
  style={
    Object {
      "backgroundColor": "var(--background-orange)",
      "boxShadow": "0 3px 2px rgba(0, 0, 0, 0.15)",
      "padding": "4rem",
    }
  }
>
  <footer
    style={
      Object {
        "color": "var(--text-color)",
        "display": "flex",
        "justifyContent": "center",
      }
    }
  >
    Â©
     
    2021
    , Made with love.
  </footer>
</section>
`;
